doc/conf.py

```diff
@@ -514,10 +514,9 @@ def js_tag_with_cache_busting(js):
             f"https://matplotlib.org/devdocs/_static/switcher.json?{SHA}"
         ),
         "version_match": (
-            # The start version to show. This must be in switcher.json.
-            # We either go to 'stable' or to 'devdocs'
-            'stable' if matplotlib.__version_info__.releaselevel == 'final'
-            else 'devdocs')
+            matplotlib.__version__
```

Comment:
should we trim this to just meso if that is all we put in our json?

Reply:
We only _display_ up to meso, but the version is still the full version, and this needs to match that for the switcher to show it right.